src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj

```diff
@@ -1,40 +1,62 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <Import Project="..\..\build\common.props" />
+
   <PropertyGroup>
     <VersionPrefix>1.0.1</VersionPrefix>
```

Comment:
Agreed. Will talk to @Eilon offline about the Sdk version. That one is much more difficult to change.